libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts

```diff
@@ -42,4 +62,50 @@ export class NxHelpAndFeedbackProvider extends AbstractTreeProvider<NxHelpAndFee
       ([title, link, icon]) => new NxHelpAndFeedbackTreeItem(title, link, icon)
     );
   }
+
+  async isConnectedToCloud(): Promise<boolean> {
+    const nxJson = (await readJsonFile(
+      join(getWorkspacePath(), 'nx.json')
+    )) as NxJsonConfiguration;
+    if (!nxJson.tasksRunnerOptions) {
+      return false;
+    }
+    return !!Object.values(nxJson.tasksRunnerOptions).find(
+      (r) => r.runner == '@nrwl/nx-cloud'
+    );
+  }
+
+  async connectToCloud() {
+    const isConnected = await this.isConnectedToCloud();
+    if (isConnected) {
+      window.showInformationMessage('You are already connected to Nx Cloud');
+      return;
+    }
+
+    window.withProgress(
+      {
+        location: ProgressLocation.Notification,
+        title: 'Connecting you to Nx Cloud...',
+      },
+      () => {
+        return new Promise((resolve) => {
+          const commandProcess = spawn(
+            getPackageManagerCommand().exec,
+            ['nx', 'connect-to-nx-cloud'],
+            { cwd: getWorkspacePath() }
+          );
+
+          commandProcess.stdout.setEncoding('utf8');
+
+          commandProcess.stdout.on('data', (data) => {
+            if (data.includes('I want faster builds')) {
+              commandProcess.stdin.write('yes\r\n', 'utf8');
+            }
+          });
```

Comment:
I'm fairly certain we can do a `--no-interaction` or some other kind of flag to automatically accept the defaults.

Reply:
I couldn't find anything like that...